src/main/java/com/jay/shoppingmall/service/handler/FileHandler.java

```diff
@@ -141,5 +132,21 @@ public String getStringImage(Image image) {
         return stringImage;
     }
 
+    public void deleteImage(Image image) {
+        String absolutePath = new File("").getAbsolutePath() + File.separator;
+        String path = absolutePath + image.getFilePath();
+
+        final File file = new File(path);
+        if (file.exists()) {
+            file.delete();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of deleting a file on this line should be checked and failure should be logged. java.io.File.delete() will simply return false if it fails to delete the file, for example, when the file does not exist. It will not throw unless there is a security exception.